AssignmentDay4/ArrOperation.java

```diff
@@ -0,0 +1,241 @@
+package AssignmentDay4;
+
+class ArrOperation {
+
+	public static int countClump(int[] arr) throws Exception {
+		/*
+		 * Return the number of clumps in the input array.Clump in an array is a
+		 * series of 2 or more adjacent elements of the same value.
+		 * 
+		 * @param arr array of input element
+		 * 
+		 * @param size size of array
+		 * 
+		 * @param count number of clump
+		 * 
+		 * @param clump number of adjacent element
+		 * 
+		 * @param itr iterator
+		 * 
+		 * @throw Exception when arr is empty.
+		 * 
+		 * @return count count of clumps.
+		 */
+		int count = 0;
+		int clump = 1;
+		int size = arr.length;
+		if (size == 0) {
+			throw new Exception("Array is empty.");
+		}
+		if (size == 1) {
+			return count;
+		}
+		for (int itr = 0; itr < (size - 1); itr++) {
+			if (arr[itr] != arr[itr + 1]) {
+				if (clump >= 2) {
+					count += 1;
+					clump = 1;
+				}
+
+			} else {
+				clump += 1;
+			}
+		}
+		if (clump >= 2) {
+			count += 1;
+			clump = 1;
+		}
+		return count;
+	}
+
+	public static int splitArray(int[] arr) throws Exception {
```

Comment:
Naming conventions not followed in the entire assignment. Logic wise it is good and working.

Reply:
She promised to not repeat this again.